qcodes_contrib_drivers/drivers/Attocube/ANC300sim.py

```diff
@@ -0,0 +1,164 @@
+              'getv 6': ['Wrong axis type','ERROR'],
+              'getv 7': ['Wrong axis type','ERROR'],
+
+              'geta 1': ['voltage = 0.000000 V'],
+              'geta 2': ['voltage = 0.000000 V'],
+              'geta 3': ['Wrong axis type','ERROR'],
+              'geta 4': ['Wrong axis type','ERROR'],
+              'geta 5': ['Wrong axis type','ERROR'],
+              'geta 6': ['Wrong axis type','ERROR'],
+              'geta 7': ['Wrong axis type','ERROR'],
+
+              'getm 1': ['mode = gnd'],
+              'getm 2': ['mode = gnd'],
+              'getm 3': ['Wrong axis type','ERROR'],
+              'getm 4': ['Wrong axis type','ERROR'],
+              'getm 5': ['Wrong axis type','ERROR'],
+              'getm 6': ['Wrong axis type','ERROR'],
+              'getm 7': ['Wrong axis type','ERROR'],
+
+              'getaci 1': ['acin = off'],
+              'getaci 2': ['acin = off'],
+              'getaci 3': ['Wrong axis type','ERROR'],
+              'getaci 4': ['Wrong axis type','ERROR'],
+              'getaci 5': ['Wrong axis type','ERROR'],
+              'getaci 6': ['Wrong axis type','ERROR'],
+              'getaci 7': ['Wrong axis type','ERROR'],
+
+              'getdci 1': ['dcin = off'],
+              'getdci 2': ['dcin = off'],
+              'getdci 3': ['Wrong axis type','ERROR'],
+              'getdci 4': ['Wrong axis type','ERROR'],
+              'getdci 5': ['Wrong axis type','ERROR'],
+              'getdci 6': ['Wrong axis type','ERROR'],
+              'getdci 7': ['Wrong axis type','ERROR'],
+
+              'gettu 1': ['trigger = off'],
+              'gettu 2': ['trigger = 2'],
+              'gettu 3': ['Wrong axis type','ERROR'],
+              'gettu 4': ['Wrong axis type','ERROR'],
+              'gettu 5': ['Wrong axis type','ERROR'],
+              'gettu 6': ['Wrong axis type','ERROR'],
+              'gettu 7': ['Wrong axis type','ERROR'],
+
+              'gettd 1': ['trigger = 1'],
+              'gettd 2': ['trigger = 3'],
+              'gettd 3': ['Wrong axis type','ERROR'],
+              'gettd 4': ['Wrong axis type','ERROR'],
+              'gettd 5': ['Wrong axis type','ERROR'],
+              'gettd 6': ['Wrong axis type','ERROR'],
+              'gettd 7': ['Wrong axis type','ERROR'],
+
+              'stepu 1': ['0'],
+              'stepu 2': ['0'],
+              'stepd 1': ['0'],
+              'stepd 2': ['0'],
+              # There is no simulation for the correct movement
+
+              }
+    
+    def __init__(self):
+        if _USE_DEBUG:
+            print("DBG-Mock: init")
+        self.closed = False
+        self.answer = []
+
+    def clear(self):
+        if _USE_DEBUG:
+            print("DBG-Mock: clear")
+
+    def close(self):
+        # make it an error to ask or write after close
+        if _USE_DEBUG:
+            print("DBG-Mock: close")
+        self.closed = True
+
+    def write(self, cmd):
+        global cmddef
```

Comment:
since cmddef is only being used as a class variable I don't think declaring it global is needed.